t/test-data/Models/hr-inclNOxExtendedWithFilters/MIGRATE_TK

```diff
@@ -0,0 +1,97 @@
+insert into public.dataset (dataset_name, dataset_pers, dataset_mod_date, dataset_version, dataset_guivariant, dataset_modelvariant, dataset_model) select dataset_name, dataset_pers, dataset_mod_date, '6.0', 'Single', 'Base', 'Agrammon6' from test.dataset where dataset_id=36958 returning dataset_id;
+insert into public.data_new (data_dataset, data_var, data_instance, data_val, data_instance_order, data_comment) SELECT 78774, data_var, data_instance, data_val, data_instance_order, data_comment FROM test.data_new WHERE data_dataset=36958;
+
```

Comment:
Hm, what is this file for?

Reply:
I had to migrate some datasets from the test database back to the production database (although we had initially agreed that this would not happen). 
Removed it from the repo, though.